app/components/layout/Footer.tsx

```diff
@@ -3,12 +3,11 @@ import Link from 'next/link';
 
 export default function Footer() {
   return (
-    <>
-      <div className="text-center pt-2 sm:-mb-4 px-10 text-light-text">
-        <span className="justify-start flex-grow">
+      <div className="linkta-footer fixed bottom-0 w-full border-t-2 pt-1 text-center text-light-text bg-[#F8F8F8]">
+        <span className="">
         ©2024 Linkta L.L.C. All rights reserved.
         </span>
-     
+
```

Comment:
I see `linkta-footer` is added here, but I don't see defined anywhere. Can it be deleted?

Reply:
@colinvandergraaf You're saying there should be no footer? There were a few places in the code where a component was commented out, or the Tailwind styles would prevent it from being displayed. The intention was not always clear to me.

Reply:
@relandboyle sorry for not being clear on this. I meant the `linkta-footer` class not the footer itself. It didn't appear to be doing anything, but not critical.

Reply:
@colinvandergraaf  Ah, I see. You're right, it's not doing anything as far as Tailwind or CSS. I added some class names just to allow me to identify the various elements. There was a lot of redundant nesting of container elements, some of which I tore out and some I left because I just got tired... I was deeply confused a few times, and giving names to things helped quite a bit.